### PR TITLE
Integrate Scp-2176

### PR DESCRIPTION
**Pull request by rayzerbrain**
Add scp-2176 breaking event and include it in other random places

### CODE REVIEW

File: Exiled.API/Enums/GrenadeType.cs [hunk 1]
```diff
@@ -29,5 +29,10 @@ public enum GrenadeType
         /// Used by <see cref="ItemType.SCP018"/>.
         /// </summary>
         Scp018,
+        
+        /// <summary>
+        /// Scp-2176 lightbulb
```
Comment (NaoUnderscore) [inline Exiled.API/Enums/GrenadeType.cs:34]: Docs should be end with a period.

File: Exiled.Events/Patches/Events/Map/BreakingScp2176.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
+// -----------------------------------------------------------------------
+// <copyright file="BreakingScp2176.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+#pragma warning disable SA1118 // The parameter spans multiple lines
```
Comment (NaoUnderscore) [inline Exiled.Events/Patches/Events/Map/BreakingScp2176.cs:8]: Move the pragma directive inside the namespace.

File: Exiled.Events/Patches/Events/Map/BreakingScp2176.cs [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            {
+                // new ExplodingGrenadeEventArgs(Player.Get(PreviousOwner.Hub), this, Array.Empty<Collider>());
+                new CodeInstruction(OpCodes.Ldarg_0),
+                new CodeInstruction(OpCodes.Ldflda, Field(typeof(Scp2176Projectile), nameof(Scp2176Projectile.PreviousOwner))),
+                new CodeInstruction(OpCodes.Ldfld, Field(typeof(Footprinting.Footprint), nameof(Footprinting.Footprint.Hub))),
+                new CodeInstruction(OpCodes.Call, Method(typeof(Exiled.API.Features.Player), nameof(Exiled.API.Features.Player.Get), new[] { typeof(ReferenceHub) })),
+                new CodeInstruction(OpCodes.Ldarg_0),
+                new CodeInstruction(OpCodes.Ldc_I4_0),
+                new CodeInstruction(OpCodes.Newarr, typeof(UnityEngine.Collider)),
+                new CodeInstruction(OpCodes.Newobj, Constructor(typeof(ExplodingGrenadeEventArgs), new[] { typeof(API.Features.Player), typeof(EffectGrenade), typeof(UnityEngine.Collider[]) })),
```
Comment (NaoUnderscore) [inline Exiled.Events/Patches/Events/Map/BreakingScp2176.cs:48]: Use `AccessTools::GetDeclaredConstructors`, it's more readable.

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]

**Review by NaoUnderscore — APPROVED** [review]

**Review by BuildBoy12 — APPROVED** [review]

**Comment by joker-119** [comment]
Fix build errors

**Pull request merged**